src/controllers/indexController.ts

```diff
@@ -2,10 +2,11 @@ import express from 'express';
 
 export const indexController = {
   index(req: express.Request, res: express.Response) {
-    console.log('Hello world received a request.');
+    console.log('Index received a request.');
```

Comment:
We can probably take out this logging unless your emotionally attached to it

Reply:
Removed 22f2730